jstp/src/test/java/com/metarhia/jstp/connection/JSTPConnectionTest.java

```diff
@@ -390,13 +394,15 @@ public void sendEscapedCharacters() throws Exception {
       @Override
       public void onConnected(boolean restored) {
         connection
-            .call("auth", "authorize", new JSArray(new Object[]{"+380962415331", "hellokitty1337"}),
+            .call("auth", "authorize",
+                new JSArray("+380962415331", "hellokitty1337"),
                 new ManualHandler() {
                   @Override
                   public void invoke(JSValue packet) {
                     JSObject userData = new JSObject();
                     userData.put("nickname", "\n\tnyaaaaaa'aaa'[((:’ –( :-)) :-| :~ =:O)],");
-                    connection.call("profile", "update", new JSArray(new Object[]{userData}),
+                    connection.call("profile", "update",
```

Comment:
I've fixed this one as it was using mixed argument-break style. 

But rules are meant to be followed strictly, either we have the rules or we don't, there is no such thing as half-rule. So as we have decided on google java code style we have to follow it.